spec.bs

```diff
@@ -82,6 +82,11 @@ spec: Fenced Frame; urlPrefix: https://wicg.github.io/fenced-frame/
 spec: Private Aggregation API; urlPrefix: https://patcg-individual-drafts.github.io/private-aggregation-api

```

Comment:
nit: I wonder if this should have `spec: private-aggregation-api` so that it isn't duplicated at the bottom in [Terms defined by reference](https://pr-preview.s3.amazonaws.com/WICG/turtledove/1297/840355a...morlovich:0ebe04d.html#index-defined-elsewhere)

Reply:
That helps a bit, though there is still kinda silly debug-details-enabled

Reply:
Hmm. So this is kinda a sideways move. It needs ;url still --- that's actually broken in shared storage, too, e.g. it tries to take you to: https://patcg-individual-drafts.github.io/private-aggregation-api/#get-batching-scope-steps
rather than to #scoping-details-get-batching-scope-steps
... and the index ends up with entries for 'enabled' and 'key' inside the private aggregation section, which is probably worse than debug-details-enabled. Like the right thing would be to import the entire type (but I guess it may not know it because it's not exported, and it's kinda a generic name so maybe shouldn't be exported?)

Reply:
Ah oof, yeah that's not ideal. Maybe the answer here is to rename it then export it? What you have seems like a pretty reasonable option at least for now, though.